packages/client/src/features/profile/ProfileForm.tsx

```diff
@@ -17,6 +18,10 @@ type ProfileFormProps = { user?: User };
 const ProfileForm: FC<ProfileFormProps> = (props: ProfileFormProps) => {
   const [updateUserInfo, { error, isSuccess }] = useUpdateUserInfoMutation();
   const updateError = error as FetchBaseQueryError;
+  const updateErrorData = updateError?.data as ErrorData;
```

Comment:
Тут, судя по updateError **?** .data, как минимум кастинг должен быть в 
```suggestion
  const updateErrorData = updateError?.data as ErrorData | undefined;
```